beta-src/src/components/ui/WDButton.tsx

```diff
@@ -1,24 +1,35 @@
 import * as React from "react";
 import Button from "@mui/material/Button";
+import WDCheckmarkIcon from "../svgr-components/WDCheckmarkIcon";
 
 interface WDButtonProps {
   children: React.ReactNode;
-  color?: "primary" | "secondary";
+  color?:
+    | "primary"
+    | "secondary"
+    | "inherit"
+    | "success"
+    | "error"
+    | "info"
+    | "warning";
```

Comment:
We should unify this with the available values in the current theme and have all these available values specified as an enum and used here and in webDiplomacyTheme palette. 

<img width="253" alt="Screen Shot 2022-02-25 at 2 10 49 PM" src="https://user-images.githubusercontent.com/93741108/155803553-61c46c94-d376-485c-969f-0ed90d5b4b49.png">